pinot-controller/src/main/resources/app/pages/Query.tsx

```diff
@@ -78,7 +79,15 @@ const useStyles = makeStyles((theme) => ({
   },
   runNowBtn: {
     marginLeft: 'auto',
-    paddingLeft: '74px',
+    paddingLeft: '10px',
```

Comment:
(not introduced in this PR)

Can you check how this looks when the window width is less? At present the buttons and the timeout field start overlapping when we reduce the width. With another button this will become a bit worse.

We can perhaps change the display prop of the divs with some media queries (I have no frontend knowledge so others can chime in)
 
![image](https://github.com/apache/pinot/assets/8644710/8f22ae12-85f3-4b40-b3f4-920196347a48)

Reply:
Sure, this seems to be something affecting the entire UI, let me work on this in an upcoming change. Thanks!

Reply:
let's at least fix this view in this PR